controllers/villains.js

```diff
@@ -0,0 +1,43 @@
+const villains = require('../models/index')
+
+const getAllVillains = async (request, response) => {
+  const listOfTeams = await villains.findAll()
+
+  return response.status(200).send(listOfTeams)
+}
+
+const getVillainBySlug = async (request, response) => {
+  const { slug } = request.params
+
+  const foundVillain = await villains.findOne({
+    where: { slug: slug }
```

Comment:
you can use the attribute which will only display certain attributes on the page.

const foundVillain = await villains.findOne({ where: { slug: slug }, attributes: ['name', 'movie', 'slug'], })